account_invoice_refund_reason/wizard/account_move_reversal.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2019 Open Source Integrators
+# Copyright (C) 2019 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class AccountMoveReversal(models.TransientModel):
+    _inherit = "account.move.reversal"
+
+    reason_id = fields.Many2one("account.move.refund.reason", string="Refund Reason")
+
+    @api.onchange("reason_id")
+    def _onchange_reason_id(self):
+        if self.reason_id:
+            self.reason = self.reason_id.name
```

Comment:
I guess we should use precompute logic

Reply:
Hello @max3903 Am i right? (I'm mentioning you because you appear as maintainer)

Reply:
I think this can be handle on another PR to add this feature?